frontend/src/views/WheresMyGene/components/HeatMap/components/Chart/index.tsx

```diff
@@ -0,0 +1,201 @@
+import { init } from "echarts";
```

Comment:
This is split out from `frontend/src/views/WheresMyGene/components/HeatMap/index.tsx`